index.html

```diff
@@ -125,6 +125,14 @@ <h2 property="schema:name">Work Items</h2>
                 </tbody>
 
                 <tbody>
+                  <tr>
+                    <td><a href="https://github.com/solid/webid-profile/blob/main/notes/pre-final-draft.md" rel="cito:citesForInformation">WebID Profile</a></td>
+                    <td><a href="https://github.com/solid/webid-profile">https://github.com/solid/webid-profile</a></td>
+                    <td>Unofficial Draft</td>
+                    <td>Ecosystem Proposal</td> 
+                    <td>Community Report</td>
```

Comment:
@VirginiaBalseiro , would you and the editors commit to that document as "Version 1.0.0" (in this PR instead of "Unofficial Draft") and indicate that in pre-final-draft.md? I realise it appears a bit strange with the current URL/filename but no big deal since you'll release the ED and 1.0.x or 1.1 soon after any way.

Reply:
I will update this PR shortly with these changes and also updating the link to the Editor's Draft.